src/test/java/test/DeadlineTest.java

```diff
@@ -0,0 +1,59 @@
+package test;
+
+import duke.Duke;
+import duke.exception.WrongCommandFormatException;
+import duke.tasktype.Deadline;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DeadlineTest {
```

Comment:
Excellent work incorporating testing into your project to ensure your project is robust!